x-pack/plugins/cases/common/types/domain/connector/latest.ts

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+export * from './v1';
+export * from './mappings.v1';
```

Comment:
in `./v1` we already do `export * from './mappings.v1';` so this looks redundant.